src/liballoc/collections/btree/navigate.rs

```diff
@@ -83,7 +86,16 @@ def_next_kv_uncheched_dealloc! {unsafe fn next_back_kv_unchecked_dealloc: left_k
 ///
 /// Safety: The change closure must not panic.
 #[inline]
-unsafe fn replace<T, R>(v: &mut T, change: impl FnOnce(T) -> (T, R)) -> R {
+fn take_mut<T>(v: &mut T, change: impl FnOnce(T) -> T) {
```

Comment:
Argh GitHub's annoying UI swallowed my comments as I attached them to the commit instead of the PR or so...

This and the next function should be `unsafe`, as they are clearly not safe to use -- they have a safety contract the caller needs to uphold.

Reply:
Rest assured I got the previous one too, and it lingers in my notifications, but I have no idea how to find it if an innocent bystander visits the issue.